abis/lemon/auxv.h

```diff
@@ -0,0 +1,9 @@
+#ifndef _ABIBITS_AUXV_H
+#define _ABIBITS_AUXV_H
+
+#define AT_PHDR 3
+#define AT_PHENT 4
+#define AT_PHNUM 5
+#define AT_ENTRY 9
```

Comment:
These definitions match the Linux ABI. You could symlink the Linux ABI's `auxv.h` instead.

Reply:
Probably better off being separate in case I need to add anything to the header in the future